packages/config/config/devices/0x027a/zen15.json

```diff
@@ -136,15 +137,15 @@
 			"allowManualEntry": false,
 			"options": [
 				{
-					"label": "Always On",
+					"label": "When device is plugged in, off otherwise",
 					"value": 0
 				},
 				{
-					"label": "On when switch is on; Off when switch is off",
+					"label": "When switch is on, off when switch is off",
 					"value": 1
 				},
 				{
-					"label": "Momentary (at status change)",
+					"label": "5 seconds on toggle, off otherwise",
 					"value": 2
```

Comment:
This is OK. You might also consider "5 seconds following switch activation, then off."